### ISSUE TITLE
[Bug]: Herd Certificates no longer trusted in Chrome after running Laravel Reverb

### ISSUE THREAD

**Issue by Nicolascanala** [beyondcode/herd-community#452]
### Operating system version

macOS Sonoma 14.3.1

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.5.0 (Build: 23)

### PHP Version

PHP 8.3

### Bug description

Google Chrome is no longer trusting my Herd-issued certificates. This happened after running Laravel Reverb and passing the hostname to the server (in order to establish a secure connection).

The problem being mentioned is "Certificate is invalid". This has happened to all my hosts secured by Herd.

Potential cause 2) I created an app and secured it with Herd. Later, I deleted the app & created it again (and ran secure again) with the same name/hostname/app-url.

It is unclear which one of these two caused the issue, as they happened at the same time.

**Comment by mpociot** [comment]
I don't think this is related to Reverb, as Reverb is not doing anything with the SSL certificates.
Could this be a browser cache issue? Are all of your SSL certificates affected by this?

Re-securing them again should fix the issue, as it generates a new certificate. So this is a bit of an odd behavior.

**Comment by Nicolascanala** [comment]
> I don't think this is related to Reverb, as Reverb is not doing anything with the SSL certificates. Could this be a browser cache issue? Are all of your SSL certificates affected by this?
> 
> Re-securing them again should fix the issue, as it generates a new certificate. So this is a bit of an odd behavior.

Thank you for your answer and for the work you've done in Herd!

The issue is now fixed, I will be closing the Github issue.

The behaviour was a little odd and I haven't been able to catch it yet. After running the reverb server, all of my certificates were rejected by Chrome. I cleared all related cache, checked the certificates themselves and it all seemed fine, yet the issue persisted. Re-started my computer, opened new chrome windows and it was still happening. 

Got frustrated and called it a day. Next morning, opened up my computer and it was all fine lol. Guess it was cache? Or maybe some kind of sabotage conspiracy come from high up? Who knows 😂  

Anyways, thanks!
